sig-autoscaling/annual-report-2024.md

```diff
@@ -30,11 +35,8 @@
       Please raise an issue in kubernetes/community, if the KEP metadata is correct but the generated list is incorrect.
 -->
 
-<!-- 
-
-
   - Stable
-    - [1610 - Container Resource based Autoscaling](https://github.com/kubernetes/enhancements/tree/master/keps/sig-autoscaling/1610-container-resource-autoscaling) - v1.30 -->
+    - [1610 - Container Resource based Autoscaling](https://github.com/kubernetes/enhancements/tree/master/keps/sig-autoscaling/1610-container-resource-autoscaling) - v1.30
```

Comment:
Should these also be listed?
- https://github.com/kubernetes/enhancements/issues/1287
- https://github.com/kubernetes/enhancements/issues/4951

Reply:
Oh, I see this is KEP work for released Kubernetes versions, ignore me.